docs/public-networks/concepts/parallel-transaction-execution.md

```diff
@@ -0,0 +1,191 @@
+transaction's list of touched addresses to the block's list, and checks for conflicts.
+In particular:
+
+1. Besu identifies conflicts by checking whether a transaction has interacted with accounts modified
+   by the block (that is, modified by previous transactions).
+2. If a conflict is detected, Besu replays the transaction using cached data or data fetched from disk.
+3. Each time a transaction is added to the block, Besu incorporates the transaction's tracked list
+   into the block's list.
+
+:::info Note
+The following are excluded from the conflict check:
+
+- Unchanged accounts read by the block.
+- Rewards given to the validator coinbase address at the end of each transaction.
+  If these were considered, every transaction would conflict with the coinbase address.
+  Besu identifies this address as a conflict only if it is accessed for reasons other than receiving
+  rewards at the transaction's conclusion.
+:::
+
+The following flowchart outlines how Besu maintains the lists of tracked addresses:
+
+<p align="center">
+
+```mermaid
+graph TD;
+  A(Start) --> B(Fetch block's touched addresses);
+  B --> C{{Check each address}};
+  C -->|Unchanged| D(Mark as read);
+  C -->|Modified| E(Add to block's tracked addresses);
+  D --> F{{Next address}};
+  E --> F;
+  F -->|More addresses?| C;
+  F -->|No more| G(Fetch transaction's touched addresses);
+  G --> H{{For each transaction address}};
+  H -->|From, sender, etc.| I(Add to transaction's tracked addresses);
+  I --> J{{Next address}};
+  J -->|More addresses?| H;
+  J -->|No more| K{{Compare block and transaction addresses}};
+  K -->|Conflict detected| L(Conflict is detected);
+  K -->|No conflict| M(Proceed with transaction);
+  L --> N(End);
+  M --> N;
+```
+
+</p>
+
+Besu's conflict detection strategy is intentionally simple to minimize edge cases.
+With this approach to parallel transaction execution,
+[approximately 40% of transactions do not require replay](#metrics).
+In the future, the conflict detection strategy may be refined to reduce false positives.
+
+You can enable parallel transaction execution using the `--Xbonsai-parallel-tx-processing-enabled` option.
+
+## Metrics
+
+Parallel transaction execution uses Besu's resources more efficiently than traditional
+sequential execution, significantly improving performance.
+
+The following metrics were collected on nodes running on Azure VMs (Standard D8as v5 – 8 vCPUs, 32
+GiB memory), with Teku and Nimbus as consensus layer (CL) clients:
+
+- **Block processing time** - With Teku as CL client, block processing time improves by at least 25%.
+  The 50th percentile decreases from 282 ms to 207 ms and the 95th
+  percentile decreases from 479 ms to 393 ms.
+
+  With Nimbus as CL client, block processing improves by approximately 45%, with the 50th percentile
+  at 155 ms, and the 95th percentile at 299 ms.
+  This is likely due to both the EL and CL clients running on the same machine, which reduces cache
```

Comment:
I think this sentence is not very clear, I suggested this wording but when I read it now, it looks not very clear. What I wanted to say is that Besu running with Nimbus has even better performances because Nimbus has less overhead on Besu, i.e less context switching and less cache misses compared to Besu running with Teku.